### ISSUE TITLE
how can i disable automatic mounting to rails routes?

### ISSUE THREAD

**Issue by mkhairi** [igorkasyanchuk/rails_db#4]
i want manually mount rails_db engine to specific routes. 
```ruby
authenticated :user, lambda {|u| u.super_user? }  do
     mount RailsDb::Engine => "/rails/db", as: 'rails_db'
end
```

**Comment by igorkasyanchuk** [comment]
I'll investigate this case, maybe some configuration will be added
But what I plan to do in near few days is to add HTTP_BASIC auth to access Rails DB console.
So you can access it with some login/password

**Comment by mkhairi** [comment]
okay, nice! :+1:

**Comment by igorkasyanchuk** [comment]
Added (see doc)

enabled - enable of disable gem (by default this gem is enabled only in development mode).
http_basic_authentication_enabled - Enable HTTP_BASIC authentication.
http_basic_authentication_user_name - HTTP_BASIC authentication user name.
http_basic_authentication_password - HTTP_BASIC authentication password.

in latest version

**Comment by mkhairi** [comment]
:+1:  for authentication!
but, i think it still need option be able on/off automatic mounting route. in some cases the are people use constraints rule in routes. :)

**Comment by igorkasyanchuk** [comment]
New version of Rails DB gem is available 0.4.

Changes:
  - autocomplete for SQL
  - ajax-ified views a little
  - data-tables to show content of your tables/sql inside your Apps (embed them)
  - option to disable automatic routes mounting

Please don't forget to share link with your friends. I need to have more feedback. Thanks

**Comment by igorkasyanchuk** [comment]
Please check now version 0.5.1, it has one more option - `verify_access_proc`, maybe it will help you too
